exercises/calculator.cpp

```diff
@@ -10,3 +10,36 @@
   Multiplication: 8
   Division: 2
 */
+
+#include <iostream>
+
+using namespace std;
+
+template<typename T>sum(T a, T b){
+	return a+b;
+}
+
+template<typename T>difference(T a, T b){
+	return a-b;
+}
+
+template<typename T>multiplication(T a, T b){
+	return a*b;
+}
+
+template<typename T>division(T a, T b){
+	if(b != 0){
+		return a/b;
+	}else{
+		return -1;
+	}
```

Comment:
Normalmente si evita il codice che non serve, in questo caso l'else non è necessario, potresti scrivere il codice in questo modo:
```C++
	if (b != 0) {
		return a / b;
	}

	return -1;
``` 

oppure.. potresti usare la condizione ternaria per rendere tutto più compatto, questione di stile 

```C++
	return b != 0 ? a / b : -1;
```